pom.xml

```diff
@@ -90,10 +90,11 @@
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <maven.compiler.source>11</maven.compiler.source>
     <maven.compiler.target>11</maven.compiler.target>
-    <dropwizard.version>2.0.17</dropwizard.version>
-    <pac4j.version>4.5.7</pac4j.version>
-    <jax-rs-pac4j.version>4.0.0</jax-rs-pac4j.version>
-    <jee-pac4j.version>5.0.0</jee-pac4j.version>
+    <dropwizard.version>4.0.0</dropwizard.version>
+    <pac4j.version>5.7.1</pac4j.version>
+    <jax-rs-pac4j.version>6.0.0-SNAPSHOT</jax-rs-pac4j.version>
```

Comment:
We need a version release of jax-rs-pac4j

Reply:
Yes, I will cut a release.